pkg/backup/actions/csi/volumesnapshot_action.go

```diff
@@ -108,12 +108,9 @@ func (p *volumeSnapshotBackupItemAction) Execute(
 	p.log.Infof("Getting VolumesnapshotContent for Volumesnapshot %s/%s",
 		vs.Namespace, vs.Name)
 
-	vsc, err := csi.WaitUntilVSCHandleIsReady(
-		vs,
-		p.crClient,
-		p.log,
-		backup.Spec.CSISnapshotTimeout.Duration,
-	)
+	ctx := context.TODO()
+
+	vsc, err := csi.GetVSCForVS(ctx, vs, p.crClient)
```

Comment:
Can we guarantee the VSC is always created when the execution comes here? Should we add the same wait-and-poll call as `WaitUntilVSCHandleIsReady`?

Reply:
In error cases, it's possible that VSC doesn't exist, but I don't think we need to add more waiting logic here.

The reason is that when the PVC BIA action fails, the VS is deleted, and it is not returned as an additional item.
We will not reach here.

If we reach here somewhat and VSC is lost, the expected behavior should be to error out.